### DISCUSSION TITLE
Emission factor calculation for "Battery Discharge"

### DISCUSSION THREAD

**Discussion by eladsimantov** [electricitymaps/electricitymaps-contrib#4532] [Q&A] [answered]
Hi.
Under the default LCA emission factors ([link](https://github.com/electricityMap/electricitymap-contrib/wiki/Emission-factors)) there is a general EF for any storage discharge of 301 [gCO2eq/kWh].
I wanted to ask how it is calculated, and which sources are used. (It says that it is a world average by Electricity Map)
Does it include LIB storage? 

Thanks!

**Comment by FelixDQ** [comment, marked as answer]
Hi @eladsimantov. We have described our methodology here: https://github.com/electricityMap/electricitymap-contrib/wiki/Carbon-intensity-calculations

Hope this answers your question :-)
